report/renderer/report-renderer.js

```diff
@@ -255,6 +256,8 @@ export class ReportRenderer {
       ));
     }
 
+    categoryRenderer.injectFinalScreenshot(categories, report.audits, scoreScale);
```

Comment:
`scoreScale` is always set... did you mean to set it only if `scoreHeader` is true?

Reply:
in the firstElementChild case, that was sometimes undefined (according to TSC), thus the if statements. but now with your better "always use a wrapper div" solution, we don't have that problem.

we still want to move the scorescale, but only if we have a final-screenshot. so.. that's why its appended first in the scoreheader, but then moved later on.